packages/react-native/React/DevSupport/RCTPausedInDebuggerOverlayController.mm

```diff
@@ -125,13 +162,36 @@ - (void)showWithMessage:(NSString *)message onResume:(void (^)(void))onResume
   [self hide];
 
   RCTPausedInDebuggerViewController *view = [[RCTPausedInDebuggerViewController alloc] init];
-#if !TARGET_OS_OSX // [macOS]
-  view.modalPresentationStyle = UIModalPresentationOverFullScreen;
   view.message = message;
   view.onResume = onResume;
+  
+#if !TARGET_OS_OSX // [macOS]
+  view.modalPresentationStyle = UIModalPresentationOverFullScreen;
```

Comment:
Are we okay with moving lines like this? Shouldn't we be wrapping this line in `#ifdefs`, and then the following block separately?